server/config/config.js

```diff
@@ -24,7 +24,7 @@ const config = {
     secret: process.env.JWT_SECRET
   },
   production: {
-    use_env_variable: process.env.DATABASE_URL,
+    use_env_variable: 'DATABASE_URL',
```

Comment:
Whats this change for?

Reply:
@seyi-adeleke heroku wasn't seeing this value, so we called it in the index file as `process.env[configObject.use_env_variable]`